common/src/main/java/org/opencds/cqf/ruler/common/utility/DaoRegistryUser.java

```diff
@@ -0,0 +1,133 @@
+package org.opencds.cqf.ruler.common.utility;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import org.hl7.fhir.instance.model.api.IBaseBundle;
+import org.hl7.fhir.instance.model.api.IBaseResource;
+import org.hl7.fhir.instance.model.api.IIdType;
+
+import ca.uhn.fhir.jpa.api.dao.DaoRegistry;
+import ca.uhn.fhir.jpa.api.dao.IFhirResourceDao;
+import ca.uhn.fhir.jpa.api.model.DaoMethodOutcome;
+import ca.uhn.fhir.jpa.searchparam.SearchParameterMap;
+import ca.uhn.fhir.rest.api.server.IBundleProvider;
+import ca.uhn.fhir.rest.api.server.RequestDetails;
+
+/**
+ * Simulate FhirDal operations until such time as that is fully baked
+ */
+public interface DaoRegistryUser {
```

Comment:
We're working through an implementation of a common API for Java FHIR access. This interface approximates that so it's easier to get convergence when that finally happens.